packages/client/src/api/forum.api.ts

```diff
@@ -182,3 +185,44 @@ export const apiGetCommentListold = () => {
   return { loading, response, error, success }
 }
 */
+
+export const apiGetEmoji = (id: string): Promise<Record<string, number> | void> =>
+  fetch(GET_EMOJI_API_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ id }),
+  })
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Network response was not ok')
+      }
+      return response.json() as Promise<Record<string, number>>
+    })
+    .catch(error => {
+      console.error(error)
+    })
+
+type addEmojiPropType = {
```

Comment:
[Предложение]
Обычно типы называют с заглавной буквы
 ```suggestion
type AddEmojiPropType = {
```

Reply:
Я привык к тому, что I***  и T*** названия используются, если тип экспортируемый, а не локальный внутри файла. Но поправлю, конечно.